src/components/v2v3/shared/SplitItem/EthAddressBeneficiary.tsx

```diff
@@ -7,16 +7,21 @@ import { isEqualAddress } from 'utils/address'
 export function ETHAddressBeneficiary({
   beneficaryAddress,
   projectOwnerAddress,
+  hideAvatar,
```

Comment:
**suggestion** maybe flip the logic from `hideAvatar` to `withEnsAvatar`, then pass that thru instead (more consistent, easier to read)